chevah/compat/tests/normal/test_capabilities.py

```diff
@@ -51,80 +45,60 @@ def test_init(self):
     def test_impersonate_local_account(self):
         """
         When running under normal account, impersonation is always False
-        on Unix and always True on Windows.
+        on Unix.
         """
         result = self.capabilities.impersonate_local_account
-        if os.name == 'posix':
-            self.assertFalse(result)
-        elif os.name == 'nt':
-            self.assertTrue(result)
-        else:
-            raise AssertionError('Unsupported os.')
+        self.assertFalse(result)
 
     def test_create_home_folder(self):
         """
         When running under normal account, we can not create home folders
         on Unix.
-
-        On Windows home folders can be created if required privileges
-        are configured for the process.
         """
         result = self.capabilities.create_home_folder
-        if os.name == 'posix':
-            self.assertFalse(result)
-        elif os.name == 'nt':
-            self.assertTrue(result)
-        else:
-            raise AssertionError('Unsupported os.')
+
+        self.assertFalse(result)
 
     def test_get_home_folder(self):
         """
         On Unix we can always get home home folder.
         On Windows, only Windows 2008 and Windows 7 can get home folder path.
         """
         result = self.capabilities.get_home_folder
-        if os.name == 'posix':
-            self.assertTrue(result)
-        elif os.name == 'nt':
-            # The Windows test is handled in elevated module.
-            pass
-        else:
-            raise AssertionError('Unsupported os.')
+
+        self.assertTrue(result)
 
     def test_getCurrentPrivilegesDescription(self):
         """
         Check getCurrentPrivilegesDescription.
         """
         text = self.capabilities.getCurrentPrivilegesDescription()
-        if self.os_family == 'posix':
-            self.assertEqual(u'root capabilities disabled.', text)
-        else:
-            # Normal Window account can impersonate.
-            self.assertContains('SeChangeNotifyPrivilege:3', text)
-            self.assertContains('SeImpersonatePrivilege:3', text)
-            self.assertContains('SeCreateGlobalPrivilege:3', text)
 
-            if self.runningAsAdministrator():
-                self.assertContains('SeCreateSymbolicLinkPrivilege:0', text)
-            else:
-                self.assertNotContains(
-                    'SeCreateSymbolicLinkPrivilege', text)
+        self.assertEqual(u'root capabilities disabled.', text)
 
     def test_pam(self):
         """
         PAM is supported on Linux/Unix with the exception of HPUX.
+        """
+        # FIXME:3813:
+        # Re-enable once we have a HP-UX buildslave.
+        # if self.os_name in ['hpux']:
+        #    self.assertFalse(self.capabilities.pam)
```

Comment:
just remove this code and provide the required details in the 3814 ticket.

there is no reason to keep commented code in the repo

please note that 3813 is about setting up the hp-ux buildslave ... but we need separate tickets to fix hp-ux support on compat / server ... etc

there is also 2703 which talks about creating a python-package for HP-UX https://github.com/chevah/python-package/pull/25